src/NuGet.Core/NuGet.Common/Utility/XmlUtility.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Xml.Linq;
+using static NuGet.Shared.XmlUtility;
+
+namespace NuGet.Common
```

Comment:
I know NuGet does this in multiple places, but another personal preference is that I don't like it when the directory and namespace do not match. If I right click the folder in VS's Solution Explorer and create a new class, it will create a class that includes the `Utility` name in the namespace, yet this class does not. In my opinion, if a project is big/complex enough to benefit from separate folders for organization, then it probably benefits from separate namespaces too.

In this case however, we're trying to keep backwards compatibility with a public API, so we can't change namespaces. In my opinion, there's no benefit to moving this file to a new `Utility` subdirectory.

Reply:
The reason I added this `XmlUtility.cs` under `Utility` folder is because the project already refers to `NuGet.Shared.XmlUtility.cs` from the project root directory.